src/guide/effects.mdx

```diff
@@ -0,0 +1,498 @@
+
+@@QtQuick.Effects.RectangularShadow is much easier to use and more efficient than
+@@QtQuick.Effects.MultiEffect, however it can only be used for rectangular shadows (as the name implies).
+On the other hand, @@QtQuick.Effects.MultiEffect can used for most common effects, including all of
+the previously mentioned effects.
+
+### Shadows
+
+The simplest way to create a rectangular drop shadow is with the @@QtQuick.Effects.RectangularShadow
+component. This component essentially creates a rectangle with blurred edges to act as a shadow. To
+use this component, you would typically create one below the source component of the shadow. To position
+a component beneath another, you can either simply create it before the other component, or use the
+@@QtQuick.Item.z property to visually position it below the other component.
+
+For example:
+
+```qml
+@@QtQuick.Rectangle {
+  id: rect
+
+  implicitWidth: 100
+  implicitHeight: 200
+  radius: 30
+}
+
+@@QtQuick.Effects.RectangularShadow {
+  anchors.fill: rect  // Fill the source rect
+  radius: rect.radius  // Have the same rounding as the source rect
+  blur: 15  // The blur/radius of the shadow
+  spread: 2  // The number of pixels from the edges of the rect the shadow extends from
+  color: "#000000"  // The colour of the shadow
+  z: -1  // Visually positions the shadow behind the source rect
+}
+```
+
+For more complex shapes, you can use the @@QtQuick.Effects.MultiEffect with the
+@@QtQuick.Effects.MultiEffect.shadowEnabled property set to `true`. You can then use the
+@@QtQuick.Effects.MultiEffect.shadowBlur, @@QtQuick.Effects.MultiEffect.shadowScale,
+@@QtQuick.Effects.MultiEffect.blurMax, @@QtQuick.Effects.MultiEffect.blurMultiplier,
+@@QtQuick.Effects.MultiEffect.shadowHorizontalOffset, @@QtQuick.Effects.MultiEffect.shadowVerticalOffset,
+@@QtQuick.Effects.MultiEffect.shadowOpacity and @@QtQuick.Effects.MultiEffect.shadowColor properties to
+tweak the drop shadow looks to your liking.
+
+Here's a simple example:
+
+> [!NOTE]
+> @@QtQuick.Effects.MultiEffect renders a new visual item alongside the source component,
+> meaning displaying the source component as well as the @@QtQuick.Effects.MultiEffect is unecessary
+> and wastes CPU cycles. To avoid this, you should either set `visible: false` or `opacity: 0` on
+> the source component. The first will prevent it from receiving mouse events while the second won't.
+
+```qml
+@@QtQuick.Rectangle {
+  id: rect
+
+  implicitWidth: 100
+  implicitHeight: 200
+  radius: 100
+  visible: false
+}
+
+@@QtQuick.Effects.MultiEffect {
+  anchors.fill: rect
+  source: rect
+  shadowEnabled: true  // This must be true for the shadow work
+  blurMax: 32  // This affects the radius of the shadow (this is also affected by `shadowBlur` and `blurMultiplier`)
+  shadowScale: 1  // This is essentially the same as `RectangularShadow#spread` but as a multiplier of the source's size
+  shadowColor: "#000000"  // The colour of the shadow
+}
+```
```

Comment:
You should use layer.effect, its easier to understand and unloads when enabled is false. Do mention that you can use it *without* layer.effect as you do here, but with is preferred.